VOSpace.tex

```diff
@@ -377,6 +377,25 @@ \subsection{Nodes and node types}
     \item .null indicates an endpoint that discards all data written to it and provides no data when read from, i.e., a bit bucket
 \end{itemize}
 
+\subsubsection{Immutable nodes}
+\label{subsubsec:immutable nodes}
```

Comment:
Spaces in a label probably work, but why take the chance?
```suggestion
\label{subsubsec:immutable-nodes}
```

Reply:
I used the space because I noticed that it is used in other parts of the documents (e.g. `\label{subsec:nodes and node types}` and `\label{sec:vospace data model}`, but I agree that using a hyphen is better.

Reply:
It seems that all the labels of the document contain spaces.

Reply:
I hadn't seen that, serves me right for not looking. The references to section labels with spaces seem to work fine, so not an issue then.